monitoring/prober/test_isa_expiry.py

```diff
@@ -11,7 +11,7 @@ def test_create(session, isa1_uuid):
   time_end = time_start + datetime.timedelta(seconds=5)
 
   resp = session.put(
-      '/identification_service_areas/{}'.format(isa1_uuid),
+      '/v1/dss/identification_service_areas/{}'.format(isa1_uuid),
```

Comment:
I would recommend not changing all of these paths to require the /v1/dss prefix as it reduces testing flexibility -- a system that includes /v1/dss can be easily tested with the previous version, but a system that does not include /v1/dss cannot be tested with this version.  The API path prefixes (/dss, /uss) are primarily meant to indicate which role would be hosting them, and not as a strict requirement on path configuration for all implementations.

Reply:
should we pass this as a flag to the test? since we need this to prefix every requests as specified in the proto